release_docs/RELEASE_PROCESS.md

```diff
@@ -96,15 +95,10 @@ For more information on the HDF5 versioning and backward and forward compatibili
     - `$ git push` 
 7. Replace "hdfgroup.github.io/hdf5/develop" in md files with "support.hdfgroup.org/releases/hdf5/vX.Y/vX.Y.Z/documentation/doxygen"
 8. ** OBSOLETE CURRENTLY **
-   Most will disappear when autotools support is removed, and none of these are currently necessary
    Update default configuration mode
-    - `$ git checkout hdf5_X_Y_Z;` and `$ bin/switch_maint_mode -disable ./configure.ac` to disable `AM_MAINTAINER_MODE`. 
+    - `$ git checkout hdf5_X_Y_Z;`. 
     - Need to set option `HDF5_GENERATE_HEADERS` to `OFF`, currently in line 996 of [src/CMakeLists.txt][u11].
-    - Change the **release preparation branch**'s (i.e. hdf5_X_Y_Z) default configuration mode from development to production in [configure.ac][u12]. 
-    - Find “Determine build mode” in [configure.ac][u12]. 
-    - Change `default=debug` to `default=production` at the bottom of the `AS_HELP_STRING` for `--enable-build-mode`.
-    - Under `if test "X-$BUILD_MODE" = X- ; then` change `BUILD_MODE=debug` to `BUILD_MODE=production`. 
-    - Run `sh ./autogen.sh` to regenerate the UNIX build system files and commit the changes. (use `git status --ignored` to see the changes and `git add -f` to add all files. First delete any new files not to be committed, notably `src/H5public.h~` and `autom4te.cache/`.)
+    - (use `git status --ignored` to see the changes and `git add -f` to add all files. First delete any new files not to be committed, notably `src/H5public.h~`.)
```

Comment:
Minor point, but this part can be probably be revised or entirely removed, since it appears to be only related to how Autotools builds modified certain files in the source directory.